frontend/app/src/theme/community/tokyonight.ts

```diff
@@ -0,0 +1,77 @@
+import type { Theme } from "../types";
+
+// Colors
+const white = "#FFFFFF";
+const offWhite = "#DFDFDF";
+const black = "#000000";

```

Comment:
This is not used so should be removed

Reply:
the `const black` is still there and needs to be removed. It doesn't really matter but it's not going pass the build with unused variables.